src/databricks/labs/ucx/source_code/linters/imports.py

```diff
@@ -303,40 +346,32 @@ def _convert_dbutils_notebook_run_to_advice(cls, node: ast.AST) -> Advisory:
         )
 
     @staticmethod
-    def get_dbutils_notebook_run_path_arg(node: ast.Call):
+    def get_dbutils_notebook_run_path_arg(node: Call):
         if len(node.args) > 0:
             return node.args[0]
         arg = next(kw for kw in node.keywords if kw.arg == "path")
         return arg.value if arg is not None else None
 
     @staticmethod
     def list_dbutils_notebook_run_calls(linter: ASTLinter) -> list[NotebookRunCall]:
-        calls = linter.locate(ast.Call, [("run", ast.Attribute), ("notebook", ast.Attribute), ("dbutils", ast.Name)])
+        calls = linter.locate(Call, [("run", Attribute), ("notebook", Attribute), ("dbutils", Name)])
         return [NotebookRunCall(call) for call in calls]
 
-    @staticmethod
-    def list_import_sources(linter: ASTLinter, problem_type: P) -> tuple[list[ImportSource], list[P]]:
+    @classmethod
+    def list_import_sources(cls, linter: ASTLinter, problem_type: P) -> tuple[list[ImportSource], list[P]]:
         problems: list[P] = []
+        sources: list[ImportSource] = []
         try:  # pylint: disable=too-many-try-statements
```

Comment:
After accepting suggestions

```suggestion
        try:
```

Reply:
Suggestion won't be accepted :-(